suite-native/message-system/src/components/FeatureMessageScreen.tsx

```diff
@@ -0,0 +1,46 @@
+import { useDispatch } from 'react-redux';
+
+import { A, G } from '@mobily/ts-belt';
+
+import { Message } from '@suite-common/suite-types';
+import { messageSystemActions } from '@suite-common/message-system';
+import { useTranslate } from '@suite-native/intl';
+
+import { MessageScreen } from './MessageScreen';
+
+type FeatureMessageScreenProps = {
+    message: Message;
+};
+
+export const FeatureMessageScreen = ({ message }: FeatureMessageScreenProps) => {
+    const dispatch = useDispatch();
+    const { translate } = useTranslate();
+    const { id: messageId, dismissible: isDismissable, category, feature } = message;
+
+    const isKillswitch =
+        feature && A.isNotEmpty(feature?.filter(item => item.domain === 'killswitch' && item.flag));
+
+    const handleDismiss = () => {
+        if (!isDismissable) return;
```

Comment:
```suggestion
```

the handleDissmiss should not be send at all if the isDismissable === false

Reply:
9860215781